flight/core/Dispatcher.php

```diff
@@ -116,7 +116,9 @@ protected function runPostFilters(string $eventName, &$output)
      */
     public function set(string $name, callable $callback): self
     {
-        if ($this->get($name) !== null) {
+        static $allowedMethods = ['notFound', 'error'];
```

Comment:
Sorry this won't work. We have to remove the whole error message because every single mapped method will throw an error.

Reply:
okay